lib/tokenize.cpp

```diff
@@ -1533,7 +1533,7 @@ void Tokenizer::simplifyTypedef()
                                 Token *tok3 = tok2->next();
 
                                 // handle missing variable name
-                                if (tok2->strAt(3) == ")" || tok2->strAt(3) == ",")
+                                if (tok2->strAt(3) == ")" || tok2->strAt(3) == "," || tok2->strAt(3) == "(")
```

Comment:
I would probably have written a Token::Match here. That is shorter code and should be faster. But I don't have a strong opinion about it so will merge anyway..